tools/fuzzer_corpus.cc

```diff
@@ -357,6 +357,7 @@ int main(int argc, const char** argv) {
   }
 
   struct stat st;
+  memset(&st, 0, sizeof(st));
```

Comment:
Out of curiosity, why is this necessary? I don’t see any suggestion in man 2 stat that stat expects the target of the pointer to have already been initialized.